components/mpas-albany-landice/cime_config/buildnml

```diff
@@ -77,6 +77,11 @@ def buildnml(case, caseroot, compname):
         grid_prefix += 'gis_1to10km_r01'
```

Comment:
Relate to comments above: Do we want to remove this?

Reply:
@mperego?

Reply:
Is this a grid we do not use? I'm fine removing it if that's the case

Reply:
Sorry @mperego -- I got confused there and didn't necessarily mean to ping you on this. You can ignore this.